hack/install_clusterlink.sh

```diff
@@ -1,4 +1,22 @@
 #!/bin/sh
+# Copyright 2024 The ClusterLink Authors.
```

Comment:
There should not be any copyright year in the file. This will fail the license check as it does not meet the expected text.
See `.licenserc.yaml` configuration at the top level